elastic/kibana
# Use recoveredCurrent and activeCurrent to determine how to update old alerts

Fixes: #180797

This PR removes execution.uuid from the alert update payload, so the recovered alerts from previous executions would not be reported again after their initial execution.

## To verify

Create a rule and let it run at least one time.
Then make one of the alerts recovered and let it run for at least 2 times,
the recovered alert should not be in the message after the first notification.

would be good to test the scenario in the following issue as well:
https://github.com/elastic/kibana/issues/164739
